website/content/en/docs/faqs/_index.md

```diff
@@ -172,3 +172,38 @@ For more information and examples, please see the type-specific docs:
 [rbac]:https://kubernetes.io/docs/reference/access-authn-authz/rbac/
 [scorecard-doc]: https://sdk.operatorframework.io/docs/testing-operators/scorecard/
 [project-doc]: /docs/overview/project-layout
+
+## Preserve the `preserveUnknownFields` in your CRDs
+
+The [`preserveUnknownFields`](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#field-pruning) will be removed if set to false when running `make bundle`. Because of some underlying data structure changes and how yaml is unmarshalled, it is best to add them back in after they have been written.
+
+You can use this script to post process the files to add the `preserveUnknownFields` back in.
+
+```sh
+function generate_preserveUnknownFieldsdata() {
```

Comment:
I @laxmikantbpandhare, 

I believe that the problem here is that users are not using the [markers](https://book.kubebuilder.io/reference/markers.html).  

So, the correct guidance seems to let them know how they use the marker properly, see (https://book.kubebuilder.io/reference/controller-gen.html):

```
crd
allowDangerousTypesboolcrdVersionsstringmaxDescLenintpreserveUnknownFieldsbooltrivialVersionsbool
generates CustomResourceDefinition objects.

``` 

Then, when they use make manifest that you will be generated and then when they use make bundle all data that is generated in the config/ dir is copied to the bundle. 

Did you try that?